UniWatch/Controllers/ClassController.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Web;
+using System.Web.Mvc;
+using UniWatch.DataAccess;
+using UniWatch.Models;
+
+namespace UniWatch.Controllers
+{
+    public class ClassController : Controller
+    {
+        private IDataAccess _dataAccess;
+
+        public ClassController()
```

Comment:
Consider chaining the constructors to stay consistent with the rest of the code.

It'll look something like this:
```
public ClassController() : this(new DataAccess.DataAccess())
{  }
```